RedHat-UX/red-hat-design-system
# docs(color-accessibility): updated color guidance

## What I did

1. Added color guidance for brand red text. 
2. Updated images
3. Updated other contrast guidance


## Testing Instructions

1. Check [color accessibility page](https://deploy-preview-1747--red-hat-design-system.netlify.app/foundations/color/accessibility/) in deploy preview
2. Compare with [Figma mockup](https://www.figma.com/design/2Gil9hfcpZYp4baKFWJLcz/Updates-to-Foundations-pages?node-id=2-100&t=iLw8iryjn2ai81NW-1)

Closes #1568
